src/main/java/gregicality/science/api/unification/materials/GCYSFirstDegreeMaterials.java

```diff
@@ -130,8 +130,9 @@ public static void init() {
                 .fluid()
                 .color(0xFFFFC8)
                 .flags(DISABLE_DECOMPOSITION)
-                .components(Hydrogen, 2, Platinum, 1, Palladium, 1, Chlorine, 6)
-                .build();
+                .components(Hydrogen, 4, Platinum, 1, Palladium, 1, Chlorine, 12)
+                .build()
+                .setFormula("H2(Pt,Pd)Cl6", true);
```

Comment:
By CEu convention commas are not to be included in formulae. Please revert this change.

Reply:
I've reverted it for now, but this is something I'd like to discuss